README.md

```diff
@@ -22,20 +22,57 @@ Use the `access_token` from Chrome to do requests
 ### Create new user
 POST /api/users/signup (*No access token in header needed for this request*)
 ```
-    {
-        "token": "<ACCESS_TOKEN_FROM_CHROME"
-    }
+{
+	"token": "<ACCESS_TOKEN_FROM_CHROME>"
+}
 ```
 returns 
-  ```
-    {
-      "data": {
-  	  	"id": <TWAIN_USER_ID>,
-    		"first_name": "<user_first_name>",
-    		"last_name": "<user_last_name>",
-    		"picture_url": "<profile_picture_url_from_google",
-    		"created_at": <CREATED_AT_TIMESTAMP>
-      },
-      "error": "<ERROR_MESSAGE>",
+```	
+{
+	"data": {
+      "id": <TWAIN_USER_ID>,
+      "first_name": "<user_first_name>",
+      "last_name": "<user_last_name>",
+      "picture_url": "<profile_picture_url_from_google",
+      "created_at": <CREATED_AT_TIMESTAMP>
+	},
+	"error": "<ERROR_MESSAGE>",
+}
```

Comment:
Why these changes? The Readme formatting looks kinda weird for this now

Reply:
Maybe remove one tab indent from all the data fields in the original version but I think you removed both

Reply:
Oh, okay, I see what you mean. For some reason it rendered differently on my computer when I was working on it earlier, but I'll definitely clean it up.